### PR TITLE
Mention the CLA: trivial marker

### PR DESCRIPTION
**Pull request by t8m**

### CODE REVIEW

**Review by mattcaswell — APPROVED** [review]

**Comment by t8m** [comment]
@mattcaswell How are the web PRs merged? Can it be done via the web interface or does it need to go through the git.openssl.org?

**Comment by mattcaswell** [comment]
> @mattcaswell How are the web PRs merged? Can it be done via the web interface or does it need to go through the git.openssl.org?

Has to be done by OMC via git.openssl.org

**Comment by Kochise** [comment]
@mattcaswell has mentionned in #11337 that :
Please amend the commit so that "CLA: trivial" has a space after ":" **and has a blank line before it**.
Moreover the 'cla: trivial' label is in lower case, so what's the right version ?

**Comment by t8m** [comment]
The bot actually accepts the marker just if it is on a line of its own, there do not have to be empty lines before or after.

**Comment by mattcaswell** [comment]
> has a blank line before it.

This is more convention than required by the bot I think i.e. we have a blank line between the subject of the commit message and the body.

**Comment by t8m** [comment]
But anyway, I've adjusted the text to mention the empty line as in CONTRIBUTING.md.

**Review by mattcaswell — APPROVED** [review]

**Comment by mattcaswell** [comment]
Needs a 2nd OMC approval

**Review by paulidale — APPROVED** [review]

File: policies/cla.html [hunk 1]
```diff
@@ -46,8 +46,10 @@
 	  <p>
 	  In practice, it is required that the author (in the git commit
 	  message) and all approving team members (in the pull request thread)
-	  agree that a change is trivial. The reviewers will normally post
-	  a statement to the effect of "I agree that it is a trivial change."
+	  agree that a change is trivial. The author has to add "CLA: trivial"
```
Comment (richsalz) [inline policies/cla.html:49]: Given the confusion that just happened, perhaps add "(note that there is a space after the colon)"

**Pull request merged**